metadata.json

```diff
@@ -59,7 +60,7 @@
     },
     {
       "name": "puppetlabs/java",
-      "version_requirement": ">= 1.0.1 < 4.0.0"
+      "version_requirement": ">= 1.0.1 < 5.0.0"
```

Comment:
not sure failing tests are related to this. But to be sure, can you rollback to `< 4.0.0`  ?

Reply:
Done.

Reply:
Still fails, because java 4 requires stdlib 5.  Looks like we need an update of java?